src/demo-app/a11y/select/select-a11y.html

```
@@ -0,0 +1,72 @@
+<section>
+  <h2>Single selection</h2>
+  <p>Select your favorite color</p>
+
+  <md-select placeholder="Colors" [(ngModel)]="selectedColor">
```

The select's `aria-label` defaults to the placeholder.